bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+            exchanged data between Web of Things.
+        </p>
+        <section>
+            <h3>Form terms</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:method</code></td>
+                        <td>Indicates the CoAP method to be used in the request</td>
+                        <td>optional</td>
+                        <td><a href="#Method">Method</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:format</code></td>
+                        <td>Indicates that a payload and a CoAP Content-Format option [[RFC7252]] be included in the request</td>
+                        <td>optional</td>
+                        <td><a href="#ContentFormat">ContentFormat</a> or array of <a href="#ContentFormat">ContentFormat</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:accept</code></td>
+                        <td>Indicates that a CoAP Accept option [[RFC7252]] be included in the request</td>
+                        <td>optional</td>
+                        <td><a href="#ContentFormat">ContentFormat</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:blockwise</code></td>
+                        <td>Indicates that a block-wise transfer is supported [[RFC7959]]</td>
+                        <td>optional</td>
+                        <td><a href="#BlockWiseTransferParameters">BlockWiseTransferParameters</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:quickblockwise</code></td>
+                        <td>Indicates that a quick block-wise transfer is supported [[I-D.ietf-core-new-block]]</td>
+                        <td>optional</td>
+                        <td><a href="#BlockWiseTransferParameters">QuickBlockWiseTransferParameters</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:hopLimit</code></td>
+                        <td>Indicates that a CoAP Hop-Limit option [[RFC8768]] be included in the request</td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+        <section id="Method">
+            <h3>Method</h3>
+            <p>
+                ... including (but not limited to) the following methods:
+            </p>
+            <table class="def" title="Method">
```

Comment:
Since this table is essentially a copy of the IANA registry, should we have it at all?